### PR TITLE
Checked for dev user in configuration/index.js

### PR DESCRIPTION
**Pull request by Omotola28**
After this i ran npm test and there is a console saying you are trying to run test with a user that is not dev

### CODE REVIEW

File: src/test/configuration/index.js [hunk 1]
```diff
@@ -3,6 +3,12 @@ const axios = require('axios');
 const { startTestServer, isStageTesting, port } = require('../../server.js');
 axios.defaults.baseURL = `http://localhost:${port}`;
 
+// Check if user is dev before running test
```
Comment (zzavidd) [inline src/test/configuration/index.js:6]: So glad you commented. Full stop at the end of comments plssss

File: src/test/configuration/index.js [hunk 2]
```diff
@@ -3,6 +3,12 @@ const axios = require('axios');
...
+if (process.env.MYSQL_USER !== 'dev') {
+  console.error('Trying to run test with user that is not dev');
+  setTimeout(() => process.exit(0));
```
Comment (zzavidd) [inline src/test/configuration/index.js:9]: Good stuff! It's simpler than I anticipated tbh 😅 
Probably don't need to prolong it with set timeout though.

File: src/test/configuration/index.js (hunk 1 above)
Comment (Omotola28) [reply src/test/configuration/index.js:6]: Cool

File: src/test/configuration/index.js (hunk 2 above)
Comment (Omotola28) [reply src/test/configuration/index.js:9]: I would remove it

**Pull request merged**